tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "strict": true,
     "types": ["jest", "node"],
     "esModuleInterop": true,
-    "rootDir": "src"
+    "rootDir": "src",
+    "lib": ["WebWorker"]
```

Comment:
According to [TS Compiler Options](https://www.typescriptlang.org/docs/handbook/compiler-options.html), the default libs injected for --target ES5 are : `DOM,ES5,ScriptHost`. Not too sure if we have to add those too if we specify a `lib` param ourselves.

Reply:
ok, will update to include them just in case